spec/unit/generator/generator_tags_spec.rb

```diff
@@ -64,7 +64,7 @@ def tags_mash_from_strings(tags_strings)
     context "with excluded and between tags" do
       let(:generator) { GitHubChangelogGenerator::Generator.new(between_tags: %w(1 2 3), exclude_tags: %w(2)) }
       it { is_expected.to be_a Array }
-      it { is_expected.to match_array(tags_mash_from_strings(%w(1 3))) }
+      it { is_expected.to match_array(tags_mash_from_strings(%w(1 2 3))) }
```

Comment:
There's no tests for `fetch_and_filter_tags` which is the most important method.

IMO the current tests are kinda false sense of security, because in fact the code is not working as expected. 

I can add tests but will take a couple weeks. It will be MUCH easier to do this in my other branch, where I've set up VCR, etc. for testing. It doesn't really make sense to add the tests in this PR.